src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -97,6 +97,15 @@ export const selectIsAccountIdEnabled = createCachedSelector(
   },
 )((_s: ReduxState, filter) => filter?.accountId ?? 'accountId')
 
+export const selectIsAnyAccountIdEnabled = createCachedSelector(
+  selectPortfolioAccounts,
+  (_state: ReduxState, accountIds: AccountId[]) => accountIds,
+  (accountsById, accountIds): boolean => {
+    if (accountIds === undefined) return false
```

Comment:
Is this typed in correctly? This should only ever be an array, and never `undefined`.

Reply:
good catch